App/src/components/settings_components/EditAvatar.tsx

```diff
@@ -9,46 +9,52 @@ const getUser = async () => {
   return response.data;
 };
```

Comment:
Can maybe put all API request functions into one file and import from there? 😃

Reply:
Yes ! I have to check that with Edo, it could be a good idea 😊